webpack.config.js

```diff
@@ -49,7 +49,6 @@ module.exports = {
       "~": path.resolve(__dirname, 'app'),
       "@shared": path.resolve(__dirname, 'shared'),
       "@server": path.resolve(__dirname, 'server'),
-      'boundless-arrow-key-navigation': 'boundless-arrow-key-navigation/build',
```

Comment:
I don't actually know what this does, I just removed all references to the library

Reply:
I believe it was a fix for ES6 imports